manifests/0000_12_kube-controller-manager-operator_01_operator.cr.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: operator.openshift.io/v1
+kind: KubeControllerManager
+metadata:
+  name: cluster
+  annotations:
+    release.openshift.io/create-only: "true"
```

Comment:
Aren't you missing replicas/logging here? I know log-level by default is 2 which should be ok but what about replicas?
@damemi